bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py

```diff
@@ -0,0 +1,91 @@
+import abc
+from typing import Callable, List, Optional, Type
+
+import xarray as xr
+from bioimageio.spec.model import nodes
+
+#: Known weigh types in order of priority
+#: First match wins
+_WEIGHT_FORMATS = ["pytorch_state_dict", "tensorflow_saved_model_bundle", "pytorch_script", "onnx"]
```

Comment:
What about `keras_hdf5`?

Reply:
untested territory...

Reply:
but I suppose we should just add it and then fix what breaks...

Reply:
We can also leave it for a follow up PR, given that we don't have the model adapter yet.